From the build; look in the `gradle_comipleTest.log` and there are compile failures like this:

```
> Task :com.ibm.ws.jpa.tests.eclipselink.query_jpa_2.1_fat:globalize NO-SOURCE
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:78: error: incompatible types: jakarta.persistence.Query cannot be converted to javax.management.Query
            Query query = em.createQuery("SELECT CEILING(?1) FROM JPA31Entity s WHERE s.intVal1 = ?2");
                                        ^
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:79: error: cannot find symbol
            query.setParameter(1, 1.1);
                 ^
  symbol:   method setParameter(int,double)
  location: variable query of type Query
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:80: error: cannot find symbol
            query.setParameter(2, 3);
                 ^
  symbol:   method setParameter(int,int)
  location: variable query of type Query
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:81: error: cannot find symbol
            query.getResultList();
                 ^
  symbol:   method getResultList()
  location: variable query of type Query
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:90: error: incompatible types: jakarta.persistence.Query cannot be converted to javax.management.Query
            query = em.createQuery("SELECT CEILING(1.1) FROM JPA31Entity s WHERE s.intVal1 = 3");
                                  ^
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:91: error: cannot find symbol
            query.getResultList();
                 ^
  symbol:   method getResultList()
  location: variable query of type Query
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:110: error: incompatible types: no instance(s) of type variable(s) T exist so that TypedQuery<T> conforms to Query
            query = em.createQuery(cquery);
                                  ^
  where T is a type-variable:
    T extends Object declared in method <T>createQuery(CriteriaQuery<T>)
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:111: error: cannot find symbol
            query.setParameter(floatParam1, 1.1f);
                 ^
  symbol:   method setParameter(ParameterExpression<Float>,float)
  location: variable query of type Query
/home/jazz_build/Build/jbe/build/open-liberty/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java:112: error: cannot find symbol
            query.setParameter(intParam1, 3);
                 ^
  symbol:   method setParameter(ParameterExpression<Integer>,int)
  location: variable query of type Query
```

The change to use `javax.management.Query` does not work; it is not compatible with `javax.persistence.Query`; the correct import needs to be restored.

--- a/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java
+++ b/dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
+ * Copyright (c) 2022,2024 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
@@ -5880,11 +5880,6 @@ public class TestMathLogic extends AbstractTestLogic {
         boolean isDerby = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DERBY);
         boolean isPostgres = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.POSTGRES);
 
-        // TODO: Disable until https://github.com/OpenLiberty/open-liberty/issues/23566 is delivered
-        if (isDB2 || isDB2Z) {
-            return;
-        }
-
         // Execute Test Case
         try {
             SQLCallListener.clearCalls();
@@ -6058,11 +6053,6 @@ public class TestMathLogic extends AbstractTestLogic {
         boolean isDB2 = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DB2LUW);
         boolean isDerby = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DERBY);
         boolean isPostgres = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.POSTGRES);
-
-        // TODO: Disable until https://github.com/OpenLiberty/open-liberty/issues/23566 is delivered
-        if (isDB2 || isDB2Z) {
-            return;
-        }
 
         // Execute Test Case
         try {
@@ -6400,11 +6390,6 @@ public class TestMathLogic extends AbstractTestLogic {
         boolean isDerby = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DERBY);
         boolean isPostgres = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.POSTGRES);
 
-        // TODO: Disable until https://github.com/OpenLiberty/open-liberty/issues/23566 is delivered
-        if (isDB2 || isDB2Z) {
-            return;
-        }
-
         // Execute Test Case
         try {
             SQLCallListener.clearCalls();
@@ -6578,11 +6563,6 @@ public class TestMathLogic extends AbstractTestLogic {
         boolean isDB2 = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DB2LUW);
         boolean isDerby = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DERBY);
         boolean isPostgres = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.POSTGRES);
-
-        // TODO: Disable until https://github.com/OpenLiberty/open-liberty/issues/23566 is delivered
-        if (isDB2 || isDB2Z) {
-            return;
-        }
 
         // Execute Test Case
         try {
@@ -6926,11 +6906,6 @@ public class TestMathLogic extends AbstractTestLogic {
         boolean isDerby = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DERBY);
         boolean isPostgres = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.POSTGRES);
 
-        // TODO: Disable until https://github.com/OpenLiberty/open-liberty/issues/23566 is delivered
-        if (isDB2 || isDB2Z) {
-            return;
-        }
-
         // Execute Test Case
         try {
             SQLCallListener.clearCalls();
@@ -7110,11 +7085,6 @@ public class TestMathLogic extends AbstractTestLogic {
         boolean isDB2 = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DB2LUW);
         boolean isDerby = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DERBY);
         boolean isPostgres = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.POSTGRES);
-
-        // TODO: Disable until https://github.com/OpenLiberty/open-liberty/issues/23566 is delivered
-        if (isDB2 || isDB2Z) {
-            return;
-        }
 
         // Execute Test Case
         try {

--- a/dev/com.ibm.ws.jpa.tests.spec10.entity_fat.common/test-applications/entity/src/com/ibm/ws/jpa/fvt/entity/testlogic/VersioningTestLogic.java
+++ b/dev/com.ibm.ws.jpa.tests.spec10.entity_fat.common/test-applications/entity/src/com/ibm/ws/jpa/fvt/entity/testlogic/VersioningTestLogic.java
@@ -88,10 +88,9 @@ public class VersioningTestLogic extends AbstractTestLogic {
         /*
          * TODO: Disabling these tests until the issue is is delivered:
          * https://github.com/OpenLiberty/open-liberty/issues/23949
-         * https://github.com/OpenLiberty/open-liberty/issues/23952
          */
         if (isDB2ZOS && JPAPersistenceProvider.ECLIPSELINK.equals(provider)
-            && (isUsingJPA21Feature() || isUsingJPA31Feature())) {
+            && (isUsingJPA21Feature())) {
             System.out.println("Skipping test; platform (" + dbProductName + ", " + dbProductVersion + ")");
             return;
         }
@@ -382,10 +381,9 @@ public class VersioningTestLogic extends AbstractTestLogic {
         /*
          * TODO: Disabling these tests until the issue is is delivered:
          * https://github.com/OpenLiberty/open-liberty/issues/23949
-         * https://github.com/OpenLiberty/open-liberty/issues/23952
          */
         if (isDB2ZOS && JPAPersistenceProvider.ECLIPSELINK.equals(provider)
-            && (isUsingJPA21Feature() || isUsingJPA31Feature())) {
+            && (isUsingJPA21Feature())) {
             System.out.println("Skipping test; platform (" + dbProductName + ", " + dbProductVersion + ")");
             return;
         }
@@ -673,10 +671,9 @@ public class VersioningTestLogic extends AbstractTestLogic {
         /*
          * TODO: Disabling these tests until the issue is is delivered:
          * https://github.com/OpenLiberty/open-liberty/issues/23949
-         * https://github.com/OpenLiberty/open-liberty/issues/23952
          */
         if (isDB2ZOS && JPAPersistenceProvider.ECLIPSELINK.equals(provider)
-            && (isUsingJPA21Feature() || isUsingJPA31Feature())) {
+            && (isUsingJPA21Feature())) {
             System.out.println("Skipping test; platform (" + dbProductName + ", " + dbProductVersion + ")");
             return;
         }

